charts/nucliadb_shared/templates/nucliadb.cm.yaml

```diff
@@ -87,4 +87,8 @@ data:
   JAEGER_QUERY_HOST: "{{ .Values.tracing.jaeger_query_host }}"
   JAEGER_QUERY_PORT: "{{ .Values.tracing.jaeger_query_http_port }}"
   JAEGER_PROPAGATION: "jaeger,b3,w3c"
-  OTEL_EXPORTER_JAEGER_AGENT_SPLIT_OVERSIZED_BATCHES: "1"
\ No newline at end of file
+  OTEL_EXPORTER_JAEGER_AGENT_SPLIT_OVERSIZED_BATCHES: "1"
+
+{{- if not .Values.telemetry.enabled }}
+  nucliadb_DISABLE_TELEMETRY: "True"
```

Comment:
What is the reason behind mixing lower and upper characters in variable name ?

Reply:
that's how the variable is declard in code ... I'm just adding it to helm charts to be able to set it on deploy

Reply:
this is the relevant line of code
https://github.com/nuclia/nucliadb/blob/main/nucliadb_telemetry/src/lib.rs#L57

Reply:
Fixed at [6d41257](https://github.com/nuclia/nucliadb/pull/515/commits/6d41257f6e0437b52203a785e85e1cdccb3990c9)